pkg/gang_schedule/interface.go

```diff
@@ -32,10 +34,10 @@ type GangScheduler interface {
 	// minNumber and select batch of pods by single or multiple label selector. The
 	// implementation should handle the relative relationship between job, pod and gang
 	// entity.
-	CreateGang(job metav1.Object, replicas map[apiv1.ReplicaType]*apiv1.ReplicaSpec) (runtime.Object, error)
+	CreateGang(job metav1.Object, replicas map[apiv1.ReplicaType]*apiv1.ReplicaSpec, schedPolicy *apiv1.SchedulingPolicy) (runtime.Object, error)
 
-	// BindPodToGang binds a object with named gang entity object.
-	BindPodToGang(obj metav1.Object, entity runtime.Object) error
+	// BindPodToGang binds an object with named gang entity object.
+	BindPodToGang(job metav1.Object, podSpec *corev1.PodTemplateSpec, gangEntity runtime.Object, rtype, index string) error
```

Comment:
the index param is not used, can be removed

Reply:
done